CONTRIBUTING.md

```diff
@@ -11,10 +11,50 @@ requirements and recommendations.
 Before you can contribute, you will need to sign the [Contributor License
 Agreement](https://identity.linuxfoundation.org/projects/cncf).
 
-## TODO
+## How to Contribute
+
+When contributing to semantic conventions, it's important to understand a few
+key, but non-obvious, aspects:
+
+- All attributes, metrics, etc. are formally defined in YAML files under
+  the `model/` directory
+- All descriptions, normative language are defined in the `docs/`
+  directory.
+  - We provide tooling to generate Markdown docuemntation from the formal
+    YAML definitons.  See [Yaml to Markdown](#yaml-to-markdown).
+  - We use Hugo to render [on our website](https://opentelemetry.io/docs/specs/semconv/).
+    You will see `<!--- Hugo front matter used to generate ...` sections
+    in markdown.  See [Hugo frontmatter](#hugo-frontmatter) for details.
+- All changes to existing attributes, metrics, etc. MUST be allowed and
+  defined in a schema file.  As part of any contribution, you should
+  include attribute changes defined in the `schema-next.yaml` file.
+  For details, please read [the schema specification](https://opentelemetry.io/docs/specs/otel/schemas/).
+- After creating a pull request, please update the [CHANGELOG](CHANGELOG.md) file with
+  a description of your changes.
+  
+Please make sure all Pull Requests are compliant with these rules!
```

Comment:
We could add a summary of these with [ ] checkboxes in a PR template.
Bumping https://github.com/open-telemetry/semantic-conventions/issues/12 for that matter 🙂